.github/workflows/ci.yml

```diff
@@ -154,3 +154,55 @@ jobs:
       - name: 'Deploy to S3: Staging'
         if: github.ref == 'refs/heads/master'
         run: aws s3 sync website s3://${{ secrets.AWS_STAGING_BUCKET_NAME }}/current --delete
+
+
+  integration-tests:
+    name: Cypress
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12
+
+      # - name: Download website
+      #   uses: actions/download-artifact@v2
+      #   with:
+      #     name: website
```

Comment:
This will be uncommented in the future. For now I just want to leave the comment for https://github.com/gnosis/gp-swap-ui/issues/219